src/adb/logcat.js

```diff
@@ -0,0 +1,10 @@
+let PassThrough = require('stream').PassThrough;
+let spawn = require('child_process').spawn;
+
+module.exports = function logcat() {
+  let proc = spawn('adb', ['logcat']);
```

Comment:
Ugh :(. How about we land the thing that assumes one device for now so that we can get this working for jsmarionette asap and then we'll do a followup for multiple device support?